tests/examples/booklending.spec.ts

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright 2021-Present The Serverless Workflow Specification Authors
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * oUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+import * as fs from 'fs';
+import {
+  actionBuilder,
+  databasedswitchBuilder,
+  delaystateBuilder,
+  eventbasedswitchBuilder,
+  eventstateBuilder,
+  oneventsBuilder,
+  operationstateBuilder,
+  transitiondataconditionBuilder,
+  transitioneventconditionBuilder,
+  workflowBuilder,
+} from '../../src';
+
+describe('booklending workflow example', () => {
+  it('should generate Workflow object', function () {
+    const workflow = workflowBuilder()
+      .id('booklending')
+      .name('Book Lending Workflow')
+      .version('1.0')
+      .start('Book Lending Request')
+      .states([
+        eventstateBuilder()
+          .name('Book Lending Request')
+          .onEvents([oneventsBuilder().eventRefs(['Book Lending Request Event']).build()])
```

Comment:
You don't need to fix but for futur work, I think it's better to stay consistant and always put the setters/build on a new line:

```typescript
.onEvents([
  oneventsBuilder()
    .eventRefs(['Book Lending Request Event'])
    .build()
  ])
```